lib/services/auth.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:brew_crew/user_id_model/user.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AuthService {
+  final FirebaseAuth auth = FirebaseAuth.instance;
+
+  User userData;
```

Comment:
nie powinieneś wystawiać userData dla wszystkich. wystaw getter.

Reply:
to wgl nie było potrzebne